exercises/sublist/example.py

```diff
@@ -1,29 +1,23 @@
-SUBLIST = 0
```

Comment:
This code seems unnecessary to me:

- in canon I see ` "expected": "equal"` which signifies a string result to me.
- therefore I deleted all references to `SUBLIST`, `SUPERLIST`, `EQUAL`, `UNEQUAL` and changed the expected results to strings.